tests/commentcounter.test.js

```diff
@@ -0,0 +1,24 @@
+import counter from '../src/module/commentcounter.js';
+import * as getc from '../src/module/getComment.js';
+
+getc.getComment = jest.fn(() => Promise.resolve([
+  { item_id: 1, username: 'meshu', comment: 'Love it seriously' },
+  { item_id: 1, username: 'meshu', comment: 'Love it seriously' },
+]));
+
+describe('Test counter function', () => {
+  it('Test counter at length 2', async () => {
+    const result = await counter(1);
+    expect(result).toBe(2);
+  });
```

Comment:
- [ ] The test is failing on this section. Kindly work on the test here so it passes. The screenshot below shows details of the error:

![image](https://user-images.githubusercontent.com/50941074/188268935-bec8c582-5028-4892-a757-e2b3e9748a1a.png)